src/lib/data/word_bank.py

```diff
@@ -19,9 +21,29 @@ def __init__(self, log_name: str = "word_bank", *args, **kwargs):
         self.load_file()
 
     @log.input()
-    def load_file(self, path: Path = Path("/usr/share/dict/words")):
-        target = [word for word in path.read_text().splitlines()]
-        self.words.extend(target)
+    def load_file(self, path: Path = None):
+        if path is None:
+            if platform.system() == "Windows":
+                # Use dictionary.txt in the user's home directory on Windows.
+                path = Path(os.path.expanduser("~")) / "dictionary.txt"
+                if not path.exists():
+                    self.log.warning(f"cannot find dictionary file: {path}")
+                    self.log.info(
+                        "Please refer to https://github.com/STAT545-UBC/Discussion/issues/49 to download the dictionary file."
```

Comment:
Thanks for finding a solution. In the future I'd like to find a platform-independent and setup-free solution (e.g. nltk), if this need (to generate random words) persists. Will accept this solution for now.